src/App.tsx

```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { v4 as uuidv4 } from 'uuid';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goodsFromServer = [
+export const goodsFromServer : string[] = [
```

Comment:
you could create an own type
etc. 

`type GoodsList = string[]`

btw, in that case it's not mandatory =)